sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Rephrase to use present tense (i.e. the pipeline that you create follow, the pipeline in this notebook use create, etc)</p><p>(i.e. we will include -&gt; we include) - make sure this applies to the whole document</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>